### PR TITLE
Content formatting

### LINKED ISSUE #1934: DOM sanitisation in document contents
Currently, the logic to display document content fields includes a `bypassSecurityTrustHtml` call. This was necessary to insert icons for named entities, I believe, but it's a security vulnerability when we allow user-created corpora.

That said, the bypass does not seem to work at the moment: content is still sanitised. (And so the icons are not displayed.) 

Related issue: we don't support safe HTML tags for documents, but because the content is included with `[innerHTML]`, the tags are not escaped. ([Actually supporting HTML fields](https://github.com/CentreForDigitalHumanities/I-analyzer/discussions/1438) might be nice but would also require that we ignore tags in Elasticsearch.)

Proposed changes:
- Change the display logic to remove the bypass call.
- Tags within a document should be escaped.
- Fix the named entity view so it shows icons again.

### PR DESCRIPTION
**Pull request by lukavdplas**
This solves some minor issues and edge cases with regard to content formatting, and adds a lot of tests. Resolves #1934

- Document preview tables now use a fixed column width, so all previews are aligned.
- Content previews show linebreaks.
- All XML/HTML tags in document contents are now escaped.
- Fixed highlights not showing up when the text contains XML/HTML tags.
- Fixed content highlighting when the content is an array.
- Whitespace in documents is now shown as-is, and not collapsed. Paragraphs are still split for main content fields.

Unrelated: fixed a typo in one of the corpus categories.

Re. whitespace, see #1954 . This update keeps the paragraph normalisation but shows whitespace within paragraphs. In a future update, I'll probably add a boolean setting in the corpus field to control whether whitespace should be preserved or normalised.

### CODE REVIEW

File: frontend/src/app/document/pipes/keyword.pipe.ts [hunk 1]
```diff
@@ -8,13 +8,9 @@ import * as _ from 'lodash';
...
-    transform(content: any): string {
-        if (_.isArray(content)) {
-            return content.map(String).join(', ');
-        } else {
-            return String(content);
-        }
-
+    transform(content: any): string[] {
+        const items = _.isArray(content) ? content : [content];
+        return items.map(String);
```
Comment (JeltevanBoheemen) [inline frontend/src/app/document/pipes/keyword.pipe.ts]: This presumes `content` is mappable to String.
What happens if it's not?
